capstone/src/main/java/com/google/sps/data/Restaurant.java

```diff
@@ -11,23 +11,27 @@
   private final double rating;
   private final double lat;
   private final double lng;
-  private final String address;
-  private final String phone;
-  private final String price;
+  private final String formattedAddress;
+  private final String formattedPhoneNumber;
+  private final String priceLevel;
   private final String website;
-  private final String ID;
+  private final String placeId;
+  private final String vicinity;
+  private final boolean openStatus;
 
-  public Restaurant(String name, int numberOfReviews, double rating, double lat, double lng, String address, String phone, String price, String website, String ID) {
+  public Restaurant(String name, int numberOfReviews, double rating, double lat, double lng, String address, String phone, String price, String website, String ID, String vicinity) {
     this.name = name;
     this.numberOfReviews = numberOfReviews;
     this.rating = rating;
     this.lat = lat;
     this.lng = lng;
-    this.address = address;
-    this.phone = phone;
-    this.price = price;
+    this.formattedAddress = address;
+    this.formattedPhoneNumber = phone;
+    this.priceLevel = price;
     this.website = website;
-    this.ID = ID;
+    this.placeId = ID;
+    this.vicinity = vicinity; 
```

Comment:
just curious but what info is vicinity?

Reply:
It is the shortened version of the address; without the state, zipcode and country.